### ISSUE TITLE
zbctl should support creating instances with startInstructions

### ISSUE THREAD

**Issue by ingorichtsmeier** [camunda/camunda#11479]
**Is your feature request related to a problem? Please describe.**
I want to start process instances with zbctl on any possible element of the process.

From the help output and looking at the source code, I found no description how to do it.

**Describe the solution you'd like**
zbctl should support the `ProcessInstanceCreationStartInstruction`. It should be added to the help output as well.

**Describe alternatives you've considered**
nothing.

**Additional context**
I'm using `zbctl 8.1.4 (commit: 8c5bc8ee)` right now.

**Comment by korthout** [comment]
Hi @ingorichtsmeier. Thanks for raising this. It was initially kept out of scope for the first iteration of this feature. Can you provide some insights into your use case? Why do you want to do this using zbctl?

**Comment by ingorichtsmeier** [comment]
Hi @korthout, I'm planning a tool/software-support to migrate running process instances from Camunda 7 to Camunda 8. As the process instance already passed the first activities, I want to continue in Camunda 8 in a later activity.

zbctl was my first approach, as I expected the complete API from https://docs.camunda.io/docs/apis-clients/grpc/ to be supported. I want to use the flexibility of the command line tool to figure out, what data are required from the running process instance and how the data should be structured to continue them most easily.

**Comment by felix-mueller** [comment]
@ingorichtsmeier can you use any of the other clients that we have?

**Comment by ingorichtsmeier** [comment]
@felix-mueller, yes, I can. 

After reading the challenges, it seems to be the fastest way to move to the Java client or get my hands dirty with Postman and gRPC, after getting an account at Postman.

**Comment by korthout** [comment]
Note that the API is also implemented in the go client. Just not in zbctl

**Comment by ingorichtsmeier** [comment]
Yeah, I've seen it already. But my Go skills are probably more limited than yours. With the challenges, this is not the perfect option to raise them here. ;-)

**Comment by ingorichtsmeier** [comment]
Hi @korthout, what about serving the start instruction in the same way as variables:

```
 --variables string         Specify variables as JSON string (default "{}")
```

`--startInstructions "[{"elementId": "activity_4"}, {"elementId": "activity_5"}]"`

**Comment by korthout** [comment]
Thanks @ingorichtsmeier. That is also a valid solution. It means that the user must write a JSON object by hand or programmatically in the shell/script. From the user's perspective, this can be an advantage when writing scripts where it is easier to build such a dynamic property, but it is harder to do in shell prompts.

From an implementation perspective, it would require parsing the JSON and extracting the relevant data to build the gRPC request with the go client. This may be harder to implement for the team than the other solution, although I'm not sure about it. It also may require adding a JSON parsing library dependency to zbctl if it doesn't already have one.
